src/cts/test/CMakeLists.txt

```diff
@@ -7,12 +7,10 @@
 or_integration_tests(
   "cts"
   TESTS
-    array
-    array_ins_delay
-    array_no_blockages
```

Comment:
why are these removed?

Reply:
They should not be removed, I added them back.